runtime/bindings/python/src/pyopenvino/graph/rt_map.cpp

```diff
@@ -20,29 +20,19 @@
 
 namespace py = pybind11;
 
-using PyRTMap = std::map<std::string, std::shared_ptr<ov::Variant>>;
+using PyRTMap = std::map<std::string, ov::Any>;
```

Comment:
This is another definition of a mapping `string -> Any`.  You've already defined it in `ngraph/core/include/openvino/core/any.hpp` in the ov namespace. Please change this alias to:
`using PyRTMap = ov::RTMap`.